kbs/src/token/jwk.rs

```diff
@@ -93,7 +93,7 @@ impl JwkAttestationTokenVerifier {
         for path in config.trusted_jwk_sets.iter() {
             match get_jwks_from_file_or_url(path).await {
                 Ok(mut jwkset) => trusted_jwk_sets.keys.append(&mut jwkset.keys),
-                Err(e) => log::warn!("error getting JWKS: {:?}", e),
+                Err(e) => bail!("error getting JWKS: {:?}", e),
```

Comment:
I am open to this, but it might be too strict to bail if we have a single JWK path that can't be reached at startup. At this point we don't know if we need these. That isn't really determined until we try to verify a token, which is where we currently bail. Anyway I can understand bailing if something is provided that can't be reached, but on the other hand maybe someone wants a config where they specify a large set of paths and don't really care if some of them fail unless they're actually needed.

wdyt @Xynnn007

Reply:
I can adjust this to bail with all the paths failed, WDYT?

Reply:
Adjusted the code to only bail in case no JWK set was found.

Mind that the loop below the check will not be executed in case the JWK set is found, and Ok() would be incorrectly returned in that case.

Reply:
I think it is better to bail ASAP at the launch time. The typical use case is a user configures all jwk paths he will use, so if any of them cannot be read it should fail to launc. It might be confusing to have partial jwk path configs that cannot work and hard to debug.

Reply:
> Anyway I can understand bailing if something is provided that can't be reached, but on the other hand maybe someone wants a config where they specify a large set of paths and don't really care if some of them fail unless they're actually needed.

that `warn()` has a bit of history because we used to have CoCo AS "trusted certs" and ITA "trusted JWK sets" combined in the same [config entry](https://github.com/confidential-containers/trustee/blob/68607d4300dda5a8ae948e2562fd06d09cbd7eca/kbs/src/token/mod.rs#L34-L39). Thanks to @Xynnn007's work these are now separated entries.

I think we want the behavior to be consistent between `trusted_jwk_sets` and `trusted_certs_paths`. The latter fails immediately if a file in the list cannot be read.

Reply:
+1 on bailing out early. If the user explicitly specifies a URL or a file which cannot be read, I'd expect the process to fail at launch instead of some attempt at graceful degradation.

Reply:
@Xynnn007, @mkulke,

I got it back to the version of bailing out early.